File: src/adn/Sequence.java
package adn;

public class Sequence
{
    public String sequence;
    public String chromosome;
    public int start;
    public int end;
   
    public Sequence next = null;
   
    public Sequence() {};
   
    public Sequence(String sequence, String chromosome, int start, int end)
    {
    this.sequence = sequence;
    this.chromosome = chromosome;
    this.start = start;
    this.end = end;
    }
   
    public String aString()
    {
        return "Sequence:" + this.sequence + "\t" + "Chr:" + this.chromosome + "\n" + "Start:" + this.start + "End:" + this.end;
    }

}

File: src/Sequence.py
class Sequence: 
        def __init__(self, sequence, chromosome,start,end):
        self.sequence = sequence
        self.chromosome = chromosome
        self.start = start
        self.end = end
        self.next = None
    
    def toString(self):
        response = "Sequence:" + self.sequence + "\tChromosome:" + self.chromosome + "\tStart:" + str(self.start) + "\tEnd:" + str(self.end)
        return response
    
    def clone(self):
        clone_ = Sequence(self.sequence,self.chromosome,self.start,self.end)
        return clone_
    
seq1 = Sequence("AACT","chr1",112,132)
print(seq1.toString())
print(seq1.clone().toString())
